seed/go-sdk/nullable/snippet.json

```diff
@@ -1,37 +1 @@
-{
```

Comment:
It's worth looking into why this is failing now - there must have been some error in the snippet generator. I bet we could see what's going wrong by introspecting [here](https://github.com/fern-api/fern/blob/104e947574915a609894487e2e245a43f7c23e49/generators/go/internal/generator/generator.go#L960).

Reply:
Ah apologies, I missed this in the diff.

Still not able to see the logs in the generated tmp file, nevertheless I've force pushed and addressed this. There is one change in the snippet diff which I'd like to discuss with you when you're back.

Reply:
Discussed offline - the snippet changes look great! Looks like this actually resolved an issue from before.